src/components/Kite/Kite.module.css

```diff
@@ -0,0 +1,88 @@
+.Kite {
+  z-index: var(--rvr-zindex-kite);
+  visibility: hidden;
+  box-sizing: border-box;
+  position: fixed;
+  top: 12px;
+  right: 12px;
+  transition: transform .6s ease-in-out;
+  animation: slide-in-right .7s;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--rvr-white);
+  width: 342px;
+  box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.2);
+  border-radius: 4px;
+  padding: 16px;
+  font-size: var(--rvr-font-size-md);
+  color: var(--rvr-color-font-dark);
+}
+
+.visible {
+  visibility: visible;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content {
+  flex: 1 1 auto;
+  min-width: 0;
+  display: flex;
+  align-items: center;
+  width: 80%;
+}
+
+.childrenWrapper {
+  margin-top: 8px;
+  margin-bottom: 8px;
+}
+
+.dismissButton {
+  flex: 0 0 auto;
+  margin-right: -16px;
+}
+
+.icon {
+  display: flex;
+  flex: 0 0 auto;
+  margin-right: 16px;
+}
+
+.enterDone {
+  opacity: 1;
+  pointer-events: visible;
+  transform: translateX(0);
+}
+
+.exit {
+  opacity: 0;
+  transform: translateX(-200px);
+}
+
+.top-right {
+  top: 12px;
+  right: 12px;
+  transition: all .6s ease-in-out;
+  transform: translateX(100%);
+}
+
+
+
+
+@keyframes slide-in-right {
+  from {
+    transform: translateX(100%);
+  }
+
+  to {
+    transform: translateX(0);
+  }
+}
```

Comment:
We should have newlines at the end of files - I think this symbol means it's missing.
Most editors have an option to add a newline automatically on save.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file